### ISSUE TITLE
AVR64DU32 Curiosity Nano Board

### ISSUE THREAD

**Issue by sstaub** [SpenceKonde/DxCore#529]
Will this board supported in future?

**Comment by SpenceKonde** [comment]
yes when DUs in general are.

**Comment by sstaub** [comment]
I bought this board some month ago directly from Microchip, also some DU 28 pin version.

**Comment by SpenceKonde** [comment]
Yes, I know the DU's are out. The DU support in the core is not ready yet.
